cg/apps/demultiplex/sample_sheet/novaseq_sample_sheet.py

```diff
@@ -88,22 +96,37 @@ def convert_sample_to_header_dict(
         sample_dict = sample.dict(by_alias=True)
         return [str(sample_dict[header]) for header in sample_sheet_headers]
 
+    def create_v2_sample_sheet_sections(self) -> List[str]:
+        """Create the header and read sections of a v2 sample sheet in a list."""
+        return [
+            SAMPLE_SHEET_HEADER,
+            SAMPLE_SHEET_HEADER_FILE_FORMAT_V2,
+            SAMPLE_SHEET_HEADER_INSTRUMENT_TYPE_NOVASEQX,  # HARD-WIRED VALUE
+            SAMPLE_SHEET_HEADER_INSTRUMENT_PLATFORM,  # HARD-WIRED VALUE
```

Comment:
As shown in the [Illumina documentation](url[)](https://support-docs.illumina.com/SHARE/SampleSheetv2/Content/SHARE/SampleSheetv2/Settings_fNV_mX.htm), these two fields are mandatory but I have not found a way to extract this info from the RunParameters